website/static/overrides.css

```diff
@@ -24,6 +41,347 @@
+  background: #f8bc45;
+}
+.pluginRowBlock .buttonWrapper:nth-of-type(2) .button {
+  background: #c596c7;
+}
+.pluginRowBlock .buttonWrapper:nth-of-type(3) .button {
+  background: #5ab3b3;
+}
+
+.homeContainer {
+  background: #1a2b34;
+}
+
+.productShowcaseSection {
+  background: #1a2b34;
+  padding-top: 50px;
+  padding-bottom: 50px;
+}
+
+.productShowcaseSection.lightBackground {
+  background: #182025;
+}
+
+.productShowcaseSection h2 {
+  text-align: left;
+  font-weight: 500;
+  margin-bottom: 1.5em;
+  font-size: 34px !important;
+}
+
+.productShowcaseSection h3 {
+  color: #f9f9f9 !important;
+  font-weight: 400;
+}
+
+.productShowcaseSection p,
+.productShowcaseSection span,
+.productShowcaseSection a,
+.productShowcaseSection code {
+  color: #f9f9f9 !important;
+  font-size: 14px;
+}
+
+.tldrSection h2,
+.tldrSection ul {
+  color: #f8bc45 !important;
+  padding: 0 !important;
+}
+
+.tldrSection h2::before {
+  content: "# ";
+}
+
+.mainContainer .tldrSection ul li {
+  list-style: none;
+  list-style-position: inside;
+  line-height: 1.0;
+}
+
+.mainContainer .tldrSection ul li a {
+  font-size: 18px;
+}
+
+.tldrSection li::before {
+  content: "* ";
+  vertical-align: sub;
+}
+
+.languagesSection h2 {
+  color: #c596c7 !important;
+}
+
+.languagesSection .accented {
+  color: #c596c7 !important;
+}
+
+.languagesSection .languageCategory p {
+  padding: 0;
+}
+
+.editorSupportSection h2 {
+  color: #5ab3b3 !important;
+}
+
+.editor {
+  padding: 0 20px 60px 0;
+  display: flex;
+  min-width: 235px;
+  flex: 1;
+  flex-grow: 0;
+  flex-basis: 22%;
+}
+
+.editor code {
+  white-space: nowrap;
+}
+
+.editorImage {
+  width: 100px;
+  height: 100px;
+}
+
+.editorInfo {
+  padding-left: 12px;
+  flex-shrink: 0;
+}
+
+.editorInfo code {
+  white-space: normal;
+}
+
+@media only screen and (min-width: 360px) and (max-width: 590px) {
+  .editor {
+    flex-direction: column;
+    min-width: 140px;
+    padding-right: 0;
+  }
+
+  .editorInfo {
+    padding-left: 0px !important;
+  }
+
+  .editorImage {
+    margin: 0;
+    width: 50px;
+    height: 50px;
+  }
+}
+
+@media only screen and (max-width: 736px) {
+  .editor {
+    padding-bottom: 30px;
+  }
+
+  .editorImage {
+    width: 75px;
+    height: 75px;
+  }
+}
+
+.getStartedSection h2 {
+  color: #e85f61 !important;
+}
+
+.getStartedSection {
+  text-align: left;
+}
+
+.getStartedSection--npm .yarnOnly {
+  display: none;
+}
+.getStartedSection:not(.getStartedSection--npm) .npmOnly {
+  display: none;
+}
+
+.getStartedSection .button {
+  border: 1px solid #e85f61;
+  color: #e85f61 !important;
+  line-height: 1em;
+}
+
+.getStartedSection .button.active {
+  background: #e85f61;
+  color: #1a2b34 !important;
+}
+
+.getStartedSection ol {
+  list-style: none;
+  color: #f9f9f9;
+  margin-top: 48px;
+  margin-left: 48px;
+}
+
+.getStartedSection ol li {
+  margin: 0 0 22px 0 !important;
+  min-height: 110px;
+  padding-left: 46;
```

Comment:
Should be 46px. Doesn't render well on windows Firefox otherwise.

Reply:
Thanks.